terraform/day1/README.md

```diff
@@ -0,0 +1,44 @@
+![TechSlate](../../global/images/ts.png)
+
+# Introduction 
+
+This page contains the links of all sub sections of day1 Terraform .
+
+## What is Terraform?
+
+- Terraform is an infrastructure as code tool that lets you build, change, and version cloud and on-prem resources safely and efficiently.
+
+- HashiCorp Terraform is an infrastructure as code tool that lets you define both cloud and on-prem resources in human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle. Terraform can manage low-level components like compute, storage, and networking resources, as well as high-level components like DNS entries and SaaS features.
+
+## What is Resource Group?
+
+- An Azure Resource Group is a logical container for resources deployed on the Microsoft Azure cloud platform. It acts as a way to manage and organize related Azure resources. Resources within a resource group can include virtual machines, storage accounts, virtual networks, web apps, databases, and more.
+
+## What is Storage Account?
+
+- An Azure Storage account is a Microsoft Azure service that provides highly scalable and secure cloud storage for various types of data. It serves as a central place to store and manage different types of data such as blobs, files, queues, tables, and disks. Azure Storage is a fundamental building block for many cloud-based applications and services.
+
+## What is variables.tf file?
+
```

Comment:
let it be variables.. instead of variables. tf...
The question can be -----------> **What are variables in Terraform ?**